### PR TITLE
DBZ-7856 Remove signal offset support for KafkaSignalChannel

### PR DESCRIPTION
**Pull request by mfvitale**
closes: https://issues.redhat.com/browse/DBZ-7856

### CODE REVIEW

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 1]
```diff
@@ -43,6 +43,14 @@ You can specify which channel is enabled by setting the `signal.enabled.channels
 The `source` channel is enabled by default, because it is required for incremental snapshot signals.
 
 
+
+=== Error handling
+
+All channels, except for the `source`, doesn't have any retry policy.
+This means that is up to you to monitor whenever a signal completed correctly or not.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/configuration/signalling.adoc:50]:
```suggestion
With the exception of the `source` channel, {prodname} signaling channels do not implement retry policies.
After you initiate a signal, be sure to verify whether it completes successfully.
```
Is there any specific external automatic monitoring mechanism that a user could implement?

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 2]
```diff
@@ -43,6 +43,14 @@ You can specify which channel is enabled by setting the `signal.enabled.channels
 The `source` channel is enabled by default, because it is required for incremental snapshot signals.
 
 
+
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/configuration/signalling.adoc:46]:
```suggestion
[id="debezium-signaling-error-handling"]
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
@mfvitale Here you go. For the second sentence, can we offer some concrete suggestion on how someone might monitor signal progress?

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 3]
```diff
@@ -43,6 +43,15 @@ You can specify which channel is enabled by setting the `signal.enabled.channels
 The `source` channel is enabled by default, because it is required for incremental snapshot signals.
 
 
+[id="debezium-signaling-error-handling"]
+=== Error handling
+
+With the exception of the `source` channel, {prodname} signaling channels do not implement retry policies.
+After you initiate a signal, be sure to verify whether it completes successfully.
+
+You can set up {link-prefix}:{link-notification}[notification] to monitor the progress of incremental/blocking snapshots.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/configuration/signalling.adoc:52]:
```suggestion
You can enable a connector to automatically report on the the progress of incremental or blocking snapshots by configuring it to send {link-prefix}:{link-notification}[notifications].
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
Oh, I should have thought of that, @mfvitale! Yes, that's perfect. I modified the text a little. If you're ok with the changes, I think we can merge this. Thanks!

File: documentation/modules/ROOT/pages/configuration/signalling.adoc (hunk 3 above)
Comment (mfvitale) [reply documentation/modules/ROOT/pages/configuration/signalling.adoc]: Done. Thanks for the suggestions.

**Review by roldanbob — APPROVED** [review]
LGTM! Thanks.

**Comment by jpechane** [comment]
@mfvitale Applied, thanks!

**Pull request merged**